### DISCUSSION TITLE
Toggle image blurring for NSFW?

### DISCUSSION THREAD

**Discussion by local306** [BlafKing/sd-civitai-browser-plus#140] [Ideas]
Is there a way to toggle the image blurring behaviour for NSFW content? Like the model or LoRA thumbnails are normal, but the example images are blurred. If there was an option to override this in the Settings portion of the extension, that would be great.

**Comment by local306** [comment]
Seems like enabling the NSFW flag in the filter menu disables the blur. Not very obvious. I still think a Settings toggle would be a little more intuitive for most people.

**Reply by BlafKing** [reply]
Not intended to be rude, but, I don't really agree, I'd say it's more hidden if it would be put in the extension settings.
Showing NSFW content is a filter, and thus it's in the filter dropdown.

Having the NSFW content checkbox turned off doesn't return any models which are tagged as NSFW, so it impacts search results.
If a model is not tagged as NSFW, but contains NSFW images as sample images, then those are blurred, which is what you encountered.

Any option that impacts the search results is put inside the filter dropdown, so that's why it's there.

Regardless, thanks for taking the time to make a post, and glad to hear that you found the solution :)
